### ISSUE TITLE
replace awesome_print with amazing_print ?

### ISSUE THREAD

**Issue by joshRpowell** [thbar/kiba-common#31]
@thbar Kiba is fantastic! Thanks for such a wonderful tool!

We're seeing this in our Ruby 2.7 bump branch: 

```
/.gem/ruby/2.7.1/gems/awesome_print-1.8.0/lib/awesome_print/formatters/base_formatter.rb:113: warning: Capturing the given block using Proc.new is deprecated; use `&block` instead
/.gem/ruby/2.7.1/gems/awesome_print-1.8.0/lib/awesome_print/inspector.rb:63: warning: Capturing the given block using Proc.new is deprecated; use `&block` instead
```

AwesomePrint is no longer receiving active updates.  ~20mo ago now. 

[AmazingPrint](https://github.com/amazing-print/amazing_print#amazing-print) is a fork and actively managed alternative.

https://github.com/thbar/kiba-common/blob/72324e08a62d95c244d0966c21914f76a64d411f/lib/kiba-common/dsl_extensions/show_me.rb#L1

Thoughts?

**Comment by thbar** [comment]
Glad you like Kiba!

It definitely makes sense. Maybe just removing the require (since the method call is the same) and adding a breaking change mention in the changelog (require either one) would be good enough.

Would you like to issue a PR with that maybe?

— Thibaut (from phone)

**Comment by thbar** [comment]
Implemented in #32. Thank you!
